### PR TITLE
More header functions

### PR DESCRIPTION
**Pull request by SuperHackio**
ActorSensorUtil - Added a bunch of IsMsg functions
ActorSwitchUtil - Added IsValid functions for the switches
ScreenUtil - Added more Wipe functions
SoundUtil - Added StartSoundPlayer

### CODE REVIEW

**Comment by shibbo** [comment]
A few issues have been noted. If they are fixed I will merge.

File: include/Util/ActorSensorUtil.h [hunk 1]
```diff
@@ -19,9 +19,67 @@ namespace MR
     bool isSensorPlayer(const HitSensor *);
     void sendMsgPush(HitSensor *, HitSensor *);
 
+	bool isMsgPlayerSpinAttack(u32);
```
Comment (shibbo) [inline include/Util/ActorSensorUtil.h:22]: indentation needs to be fixed here

File: include/Util/ActorSwitchUtil.h [hunk 2]
```diff
@@ -16,4 +16,9 @@ namespace MR
     bool useStageSwitchReadDead(LiveActor *, const JMapInfoIter &);
 
     void syncStageSwitchAppear(LiveActor *);
+
+	bool isValidSwitchA(const LiveActor *);
```
Comment (shibbo) [inline include/Util/ActorSwitchUtil.h:20]: indentation needs to be fixed here

File: include/Util/ScreenUtil.h [hunk 3]
```diff
@@ -4,5 +4,25 @@
 
 namespace MR
 {
-    void closeSystemWipeFadeWithCaptureScreen(s32);
+	void openSystemWipeCircle(s32 Fadetime); // Located at 0x80058C00 in USA. I don't want to change the symbol map
```
Comment (shibbo) [inline include/Util/ScreenUtil.h:7]: indention is mismatched here

File: include/Util/ScreenUtil.h [hunk 4]
```diff
@@ -4,5 +4,25 @@
...
+	void closeSystemWipeCircleWithCaptureScreen(s32 Fadetime);
+    void closeSystemWipeFadeWithCaptureScreen(s32 Fadetime);
+	// Activate the Game Over wipe
+	void startGameOverWipe();
+	// Activate the Bowser wipe used when the player dies
+	void startDownWipe();
+
+	void openSystemWipeMario(s32 Fadetime);
+	void closeSystemWipeMario(s32 Fadetime);
+
```
Comment (shibbo) [inline include/Util/ScreenUtil.h:20]: only have one empty space if there are gaps between known functions

File: include/Util/SoundUtil.h [hunk 5]
```diff
@@ -7,5 +7,7 @@ class LiveActor;
 namespace MR
 {
     void startSound(const LiveActor *, const char *, s32, s32);
+	//void startSound(const LiveActor *, JAISoundID, s32, s32); //This is a thing I just have no idea what JAISoundID is
     void startLevelSound(const LiveActor *, const char *, s32, s32, s32);
+	void startSoundPlayer(const char *, s32, s32);
```
Comment (shibbo) [inline include/Util/SoundUtil.h:12]: indentation

File: include/Util/SoundUtil.h [hunk 6]
```diff
@@ -7,5 +7,7 @@ class LiveActor;
 namespace MR
 {
     void startSound(const LiveActor *, const char *, s32, s32);
+	//void startSound(const LiveActor *, JAISoundID, s32, s32); //This is a thing I just have no idea what JAISoundID is
```
Comment (shibbo) [inline include/Util/SoundUtil.h:10]: Only comment out functions that have complete types. JAISoundID, in this instance, is a typdef'd variable.

File: include/Util/ActorSensorUtil.h [hunk 7]
```diff
@@ -19,9 +19,67 @@ namespace MR
...
+	bool isMsgPlayerHipDrop(u32);
+
     bool isMsgPlayerUpperPunch(u32);
 
+	bool isMsgPlayerKick(u32);
+
     bool isMsgPlayerHipDropFloor(u32);
 
     bool isMsgJetTurtleAttack(u32);
+
```
Comment (shibbo) [inline include/Util/ActorSensorUtil.h:35]: do not have spaces between known functions

**Comment by SuperHackio** [comment]
Should all be fixed now

**Comment by shibbo** [comment]
Looks good to me. Merging.

**Pull request merged**